tools/niminst/makefile.nimf

```diff
@@ -157,6 +157,9 @@ endif
 ifeq ($(ucpu),armv7hl)
   mycpu = arm
 endif
+ifeq ($(ucpu),armv8l)
+  mycpu = arm64
```

Comment:
Are you sure that it should be treated like arm64? It's rather weird to see toolchain reporting `armv8l`, I'm not sure if this means its an armv8 cpu in 32-bit mode or 64-bit mode. From this issue https://github.com/randombit/botan/issues/1543#issuecomment-382186505 it looks like it should be 32-bit, but I'm not sure.

Reply:
ARMv8 is native at 64bit but can also run in 32bit mode (for backward compatibility)

Reply:
Yea but we are detecting using `uname -m` and `armv8l` seems to be what is returned when the kernel is 32-bit.

So I'm unsure if we should default to 64-bit here. Do toolchains running on these systems target aarch64?

Reply:
The arm64 where I'm seeing the build fail is identifying as armv8l: https://buildd.debian.org/status/fetch.php?pkg=nim&arch=armhf&ver=1.4.8-3&stamp=1629381474&raw=0

Reply:
Isn't that build targeting armhf, which is 32-bit?

Reply:
It is, but I'm not sure if we wouldn't get armv8l also when targeting arm64 or not.

Reply:
Given that 64bit arm is pretty much standardized as aarch64 everywhere, I think it's safe to assume that armv8l is marker for aarch32, esp when that's what debian yields

Reply:
Right! Thank you! I updated the PR.
This helps: https://bgamari.github.io/posts/2019-06-12-arm-terminology.html